internal/digest/sha384/sha384.go

```diff
@@ -53,7 +52,7 @@ func (h *Hash) Digest() *digest.Digest {
 func (h *Hash) AddBytes(b []byte) error {
 	_, err := h.hash.Write(b)
 	if err != nil {
-		return errors.Wrap(err, "writing to hash stream failed")
+		return fmt.Errorf("writing to hash stream failed")
```

Comment:
```suggestion
		return fmt.Errorf("writing to hash stream failed: %w", err)
```
wrapping the error is missing here